docs/user-guide/rug/commands.md

```diff
@@ -1,6 +1,6 @@
 Controlling communication and information flows is paramount to
```

Comment:
I'm heavily reworking this section (nearly ready for PR), so it'll be hard to merge these when I do...